Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -226,5 +365,520 @@ public override bool IsServerConnected()
         {
             throw new NotImplementedException();
         }
+
+        public QC.QCTestSet ImportTestSetData(QC.QCTestSet testSet)
+        {
+            QCTestInstanceColl testInstances = GetTestsFromTSId(testSet.TestSetID);
+
+            foreach (QCTestInstance testInstance in testInstances)
+            {
+                testSet.Tests.Add(ImportTSTest(testInstance));
+            }
+
+            return testSet;
+        }
+
+        public QCTestInstanceColl GetTestsFromTSId(string testSetID)
+        {
+            try
+            {
+                List<TestSuite_Test_Link> TSLink = new List<TestSuite_Test_Link>();
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'TSLink'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6317725)